bin/filter_gtf.py

```diff
@@ -20,16 +19,24 @@ def extract_fasta_seq_names(fasta_name: str) -> Set[str]:
         return {line[1:].split(None, 1)[0] for line in fasta if line.startswith(">")}
 
 
-def tab_delimited(file: str) -> float:
-    """Check if file is tab-delimited and return median number of tabs."""
+def tab_checks(file: str) -> (bool, bool):
```

Comment:
This function doesn't need to be changed (see below), and you've changed the existing logic- could you revert please?